fdbserver/include/fdbserver/DDRelocationQueue.h

```diff
@@ -20,6 +20,7 @@
 #ifndef FOUNDATIONDB_DDRELOCATIONQUEUE_H
 #define FOUNDATIONDB_DDRELOCATIONQUEUE_H
 
+#include <numeric>
```

Comment:
Is this include intentional?

Reply:
Yes. This `include` exists in `main` branch and fixes a compilation problem.